packages/@react-spectrum/progress/test/ProgressBar.test.js

```diff
@@ -92,4 +92,22 @@ describe('ProgressBar', function () {
     let progressBar = getByTestId('test');
     expect(progressBar).toBeInTheDocument();
   });
+
+  it('changes class with static color prop', () => {
```

Comment:
this is covered by the chromatic stories, we try not to do class name tests as much as possible